release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -132,6 +132,33 @@ For more information, see xref:../installing/installing_bare_metal_ipi/ipi-insta
 [id="ocp-4-11-web-console"]
 === Web console
 
+[id="ocp-4-11-dynamic-plug-in-updates"]
+==== Dynamic Plug-in updates
+
+With this update, you can now enable and disable plug-ins in the *Administration* -> *Cluster Settings* page. You can also add different extension types to customize your plug-in. 
+
+For more information, see xref:../web_console/dynamic-plug-ins.adoc#dunamic-plug-ins_web_console[Dynamic plug-ins].
+
+[id="ocp-4-11-dark-mode"]
+==== Support for dark mode theme
+
+The {product-title} web console now supports the dark mode theme. On the *User Settings* page, select your preferred mode to view the web console in.
+
+[id="ocp-globally-installed-operators-display-operand-instances"]
+==== Display operand instances for all managed namespaces on the *Operator Details* page
+
+When viewing the *Installed Operators* page set to `All projects` and then selecting an Operator that is available in the globally installed `All namespaces` to view the details of that Operator in installed namespace.
```

Comment:
Long sentence, and I'm finding it difficult to understand the message. Consider breaking up the sentence into smaller sentences for clarity and accuracy.

Reply:
+1 I can't follow this either